lightning/src/routing/router.rs

```diff
@@ -165,14 +165,22 @@ pub struct RouteParameters {
 
 	/// The CLTV on the final hop of the failed payment path.
 	pub final_cltv_expiry_delta: u32,
+
+	/// The maximum total CLTV delta we accept for the route.
+	pub max_total_cltv_expiry_delta: u32,
 }
 
 impl_writeable_tlv_based!(RouteParameters, {
 	(0, payee, required),
 	(2, final_value_msat, required),
 	(4, final_cltv_expiry_delta, required),
+	(6, max_total_cltv_expiry_delta, required),
```

Comment:
I think we'll want to make this an odd type (`1`). But we can use `(default_value, DEFAULT_MAX_TOTAL_CLTV_DELTA)` instead of `required`.

Reply:
Done in https://github.com/lightningdevkit/rust-lightning/pull/1234/commits/b7cb0f3eca8f491de09c97e41b8f090c3d27264e.